CHANGELOG.rst

```diff
@@ -1,5 +1,51 @@
 .. _changelog:
 
+0.23.0
+------
+
+Renku ``0.23.0`` introduces the Renku Workflow File, a friendlier way to encode and run your data analysis pipelines
+on Renku. This yaml-based file is easy to write and execution is as easy as `renku run workflow.yml`. Add a workflow
+to your Renku project with our docs [here](https://renku.readthedocs.io/en/stable/topic-guides/workflows/workflow-file.html)!
+
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**🌟 New Features**
+
+ * 📝 **Renku CLI**: Add support for writing workflow files which can be executed directly, greatly improving the UX
+   around workflows
+   (`#3176 <https://github.com/SwissDataScienceCenter/renku-python/pull/3176>`_)
+ * **Knowledge Graph**: Partial support for Workflow File metadata.
```

Comment:
I don't understand this - would either of these make sense? 

Add partial support for indexing workflows? 
Add partial support for processing workflow metadata?  (this is not really much better tbh in the sense that what this processing is is not clear at all...)

Reply:
Yes, I see your point. Should we even drop this line altogether? From the user's point of view, it's crucial if things will work or not as it's not even possible that he/she will be able to see the difference between the so-called 'partial support' and full support if there is such. I mean, for the time being, the user wouldn't have access to the full-picture data even if it's there.